frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/modelServingDeploy.cy.ts

```diff
@@ -755,6 +775,512 @@ describe('Model Serving Deploy Wizard', () => {
     });
   });
 
+  it('Do not deploy KServe model when user cannot edit namespace (only one serving platform enabled)', () => {
+    // If only one platform is enabled, project platform selection has not happened yet and patching the namespace with the platform happens at deploy time.
+    initIntercepts({
+      modelType: ServingRuntimeModelType.PREDICTIVE,
+      rejectAddSupportServingPlatformProject: true,
+    });
+    cy.interceptK8sList(
+      { model: InferenceServiceModel, ns: 'test-project' },
+      mockK8sResourceList([mockInferenceServiceK8sResource({})]),
+    );
+    cy.interceptK8sList(
+      { model: ServingRuntimeModel, ns: 'test-project' },
+      mockK8sResourceList([mockServingRuntimeK8sResource({})]),
+    );
+
+    // TODO: visit directly when plugin is enabled
+    cy.visitWithLogin(
+      '/ai-hub/deployments/test-project?devFeatureFlags=Model+Serving+Plugin%3Dtrue',
+    );
```

Comment:
The model serving refactor plugin is enabled, so we can use the proper visit function
```suggestion
    modelServingGlobal.visit('test-project');
```